src/entity/Notification.ts

```diff
@@ -1,18 +1,81 @@
-import { Column, Entity, PrimaryColumn } from 'typeorm';
+import {
+  Column,
+  Entity,
+  Index,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from './User';
+import { NotificationAvatar } from './NotificationAvatar';
+import { NotificationAttachment } from './NotificationAttachment';
+
+export type NotificationType =
+  | 'community_picks_failed'
+  | 'community_picks_success'
```

Comment:
Should this not be `community_picks_succeeded` to be inline with correct tense?

Reply:
Copied from the DR. I'll fix that!